pkgs/development/python-modules/ansible-runner/default.nix

```
@@ -30,8 +31,10 @@ buildPythonPackage rec {
     six
   ];
 
+  # test_process_isolation_settings is currently broken on Darwin Catalina
+  # https://github.com/ansible/ansible-runner/issues/413
   checkPhase = ''
-    HOME=$(mktemp -d) pytest --ignore test/unit/test_runner.py -k "not test_prepare"
+    HOME=$(mktemp -d) pytest --ignore test/unit/test_runner.py -k "not test_prepare${lib.optionalString stdenv.isDarwin " and not test_process_isolation_settings"}"
```

all tests have to be prefixed with test_, so it's not very useful to identify tests. And I think i would prefer the whitespace to be around the entry
```suggestion
    HOME=$TMPDIR pytest \
      --ignore test/unit/test_runner.py \
      -k "not prepare ${lib.optionalString stdenv.isDarwin "and not process_isolation_settings"}"
```

---

ok trying your suggestion.